contracts/modules/capital/CowSwapOperator.sol

```diff
@@ -0,0 +1,256 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.9;
+
+import "../../external/cow/GPv2Order.sol";
+import "@openzeppelin/contracts-v4/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts-v4/utils/math/Math.sol";
+import "../../interfaces/INXMMaster.sol";
+import "../../interfaces/IPool.sol";
+import "../../interfaces/IWeth.sol";
+import "../../interfaces/ICowSettlement.sol";
+import "../../interfaces/IPriceFeedOracle.sol";
+
+contract CowSwapOperator {
+  // Storage
+  ICowSettlement public immutable cowSettlement;
+  address public immutable cowVaultRelayer;
+  INXMMaster public immutable master;
+  address public immutable swapController;
+  IWeth public immutable weth;
+  IPriceFeedOracle public immutable priceFeedOracle;
+  bytes public currentOrderUID;
+  bytes32 public immutable domainSeparator;
```

Comment:
Immutable variables do use the storage, they're replaced in the contract bytecode directly (similar to constants). Move them to a separate "section" (storage, immutables, constants, events, etc)